src/Avalonia.Controls/Window.cs

```diff
@@ -158,6 +168,16 @@ public bool HasSystemDecorations
         }
 
         /// <summary>
+        /// Sets if the Window should Covert the taskbar when Maximized. Only applies to Windows 
```

Comment:
Covert? As in covert operations? ;) Also nit: we don't capitalize "Window" elsewhere, unless we're doing a `<see cref="Window">`